NOTICE.txt

```diff
@@ -2375,11 +2375,11 @@ Contents of probable licence file $GOMODCACHE/github.com/elastic/go-licenser@v0.
 
 --------------------------------------------------------------------------------
 Dependency : github.com/elastic/go-sysinfo
-Version: v1.15.0
+Version: v1.15.1
 Licence type (autodetected): Apache-2.0
 --------------------------------------------------------------------------------
 
-Contents of probable licence file $GOMODCACHE/github.com/elastic/go-sysinfo@v1.15.0/LICENSE.txt:
+Contents of probable licence file $GOMODCACHE/github.com/elastic/go-sysinfo@v1.15.1/LICENSE.txt:
```

Comment:
The rest of the changes here is just a clean-up of the line endings, done by the fix in `notice.go`, you can review it by omitting whitespace from diff
https://github.com/elastic/elastic-agent/pull/6866/files?diff=split&w=1